<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="dicegifLabel">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>150</y>
     <width>714</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="monsterLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>800</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="healthpointBar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="rollButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>640</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>ROLL! (3)</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="maintextBrowser">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>800</width>
     <height>120</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;We get text here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstdiceLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="seconddiceLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="thirddiceLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>640</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fourthdiceLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>640</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fifthdiceLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>640</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="firstdiceButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>730</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seconddiceButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>730</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thirddiceButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>730</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourthdiceButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>730</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fifthdiceButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>730</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fix!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="killButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>즉사버튼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>720</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAY!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
